docs/source/models.rst

```diff
@@ -4440,3 +4440,200 @@ Refer to the `sktime mlflow documentation <https://www.sktime.net/en/latest/api_
 
     print(loaded_model.predict())
     print(loaded_pyfunc.predict(pd.DataFrame()))
+
+MLflavors
```

Comment:
I propose to move this section to a separate doc page with header "third-party MLflow flavor integration", WDYT ? Because in this page we usually put mlflow builtin API docs.

Reply:
I guess from content perspective it makes sense to have it part of the Models docs page, but agree it might be better to move it to a separate page to make it more explicit that these are third-party integrations

Happy to change the docs as per your proposal as part of this PR

Reply:
@WeichenXu123 I actually named it "Community Model Flavors" consistent with terminology in current [table of content](https://mlflow.org/docs/latest/models.html#mlflow-models) - let me know if you prefer to use the one from above, then I will change it accordingly